File: src/WebClient/WareHouse/src/app/entity/AuditCouncil.ts
import { Inward } from './Inward';
import { BaseEntity } from "./BaseEntity";
import { WareHouseItem } from "./WareHouseItem";
import { Audit } from './Audit';
export interface AuditCouncil extends BaseEntity {
    auditId: string;
    employeeId: string;
    employeeName: string;
    role: string;
    audit: Audit;
}

File: src/Services/WareHouse/WareHouse.Domain/Entity/AuditCouncil.cs
using System;
using System.Collections.Generic;
using Share.BaseCore.BaseNop;
#nullable disable

namespace WareHouse.Domain.Entity
{
    public partial class AuditCouncil : BaseEntity
    {
        public string AuditId { get; set; }
        public string EmployeeId { get; set; }
        public string EmployeeName { get; set; }
        public string Role { get; set; }

        public virtual Audit Audit { get; set; }
    }
}
